feat: add devops folder icon

alexanderdombroski opened the pull request:
# Description

Added a folder icon for devops using the same colors to match the `folder-ci.svg`

## Contribution Guidelines

- [X] By creating this pull request, I acknowledge that I have read the [Contribution Guidelines](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md) for this project
- [X] I have read the [Code Of Conduct](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CODE_OF_CONDUCT.md) and promise to abide by these rules

alexanderdombroski committed:
figure 8

alexanderdombroski committed:
sideways 8

alexanderdombroski committed:
circle

alexanderdombroski committed:
true circle

alexanderdombroski committed:
two circles

alexanderdombroski committed:
true direction figure 8

alexanderdombroski committed:
glasses

alexanderdombroski committed:
rough devops

alexanderdombroski committed:
expand rough pointy devops

alexanderdombroski committed:
devops final

alexanderdombroski committed:
minify svg

alexanderdombroski committed:
add devops folder svg mapping

alexanderdombroski committed:
Merge branch 'main' into main

PKief reviewed (changes requested):
Hi @alexanderdombroski thanks for contributing. I can understand the idea behind that motive, but I', afraid that on small resolution it has more similarities to some glasses than the devops topic. I would recommend that we choose another motive here which will fit better to a devops folder name.

SayanShankhari commented:
this is what i was thinking to solve this, the pair of color might avoid looking like glass

<img width="80" height="80" alt="inf" src="https://github.com/user-attachments/assets/29dc6985-1888-4b29-a1bf-85879711d088" />

alexanderdombroski committed:
Merge branch 'material-extensions:main' into main

alexanderdombroski committed:
feat: use more detailed devops motive

Co-authored-by: Sayan Shankhari <sayan0shankhari@gmail.com>

alexanderdombroski commented:
Thanks @SayanShankhari, I coauthored you to give you credit. Love the more detailed motive! I adjusted it to fit into the 32 32 viewbox and removed the ink metadata to slim it down.

SayanShankhari commented:
thanks!
just to inform you, according to the docs and PKief, the default vs code icon size is 16x16, we can use that one only, don't know why some choose 32x32,
it's vector, so no worries, let's see if it gets approved

i would also request you to add dev-ops folder name

```js
      {
        name: 'folder-devops',
        folderNames: ['devops', 'dev-ops'],
      }
```
:)

PKief committed:
Merge branch 'main' into main

alexanderdombroski committed:
feat: add devops name variations

alexanderdombroski commented:
ok I added more folder name variations. I included some for DevSecOps too, as most of the unofficial logos are nearly the same with DevOps, just with an added "sec" above the infinity sign. Ready for review!

SayanShankhari commented:
update, color inverted for the infinite parts, note earlier logo was not showing rightmost bottom as light blue is not distinguishable in light theme.

<img width="80" src="https://github.com/user-attachments/assets/3d238e46-9a71-496b-bd56-bd5f37cfd930" />

alexanderdombroski committed:
Merge branch 'material-extensions:main' into main

alexanderdombroski committed:
feat: invert folder devops svg colors

alexanderdombroski commented:
I updated the svgs to use your inverted colors and 16 16 viewbox

SayanShankhari commented:
great, thanks,
somehow i can't see "created preview" image in github actions section
